packages/@cdktf/provider-generator/lib/get/__tests__/generator/__snapshots__/resource-types.test.ts.snap

```diff
@@ -5550,6 +5550,600 @@ export class CloudfrontDistribution extends cdktf.TerraformResource {
 "
 `;
 
+exports[`generate a datadog spans metric 1`] = `
+"// https://registry.terraform.io/providers/datadog/datadog/latest/docs/resources/spans_metric
+// generated from terraform resource schema
+
+import { Construct } from 'constructs';
+import * as cdktf from 'cdktf';
+
+// Configuration
+
+export interface SpansMetricConfig extends cdktf.TerraformMetaArguments {
```

Comment:
As described in the issue, this interface should contain the `compute` attribute.